en/components/list.md

```diff
@@ -514,6 +516,8 @@ Finally, we need to apply the filtering pipe to our contacts data before we can
 </igx-list-item>
 ```
 
+Looking for more filtering options? Check out our angular filtering options, including [**Excel Style Filtering**](grid/excel_style_filtering.md) and [**Advanced Filtering Options**](grid/advanced_filtering.md).
```

Comment:
I don't think this paragraph is ok to be here, since the excel style filtering and the advanced filtering are in the context of the grid, not the list.